src/main/java/frc/robot/Robot.java

```diff
@@ -3,12 +3,24 @@
 import com.flash3388.flashlib.frc.robot.FrcRobotControl;
 import com.flash3388.flashlib.frc.robot.base.iterative.DelegatingFrcRobotControl;
 import com.flash3388.flashlib.frc.robot.base.iterative.IterativeFrcRobot;
+import com.flash3388.flashlib.hid.XboxButton;
+import com.flash3388.flashlib.hid.XboxController;
+import frc.robot.actions.TakeOut;
+import frc.robot.subsystems.Intake;
 
 public class Robot extends DelegatingFrcRobotControl implements IterativeFrcRobot {
+    private Intake intake;
+    private XboxController xbox;
 
-    public Robot(FrcRobotControl robotControl) {
+    public Robot(FrcRobotControl robotControl)
+    {
         super(robotControl);
+        this.intake = SystemFactory.createIntake();
+        this.xbox = getHidInterface().newXboxController(RobotMap.XBOX);
+       // xbox.getButton(XboxButton.B).whenActive(new TakeOut(intake));
```

Comment:
Add a button for both `TakeOut` and `TakeIn`. As we want to control both.

For now, because the sensor code isn't added yet, configure the buttons to be `whileActive`, so that when we release the button, it will stop.